v2/passwordless/serverless/with-aws-lambda/auth-serverless.mdx

```diff
@@ -5,16 +5,27 @@ hide_title: true
 ---
```

Comment:
remove line -> https://github.com/supertokens/docs/blob/feat/lambda-integration/v2/session/serverless/with-aws-lambda/frontend.mdx?plain=1#L5

Reply:
Also, change the text in the file to "Visit the [frontend guide](../../quick-setup/frontend) to finish the frontend setup."

Reply:
Both of these should only happen for the session recipe

Reply:
Done.